src/components/policy/policy_external/test/sql_pt_representation_test.cc

```diff
@@ -401,7 +406,7 @@ TEST_F(SQLPTRepresentationTest,
   // Check PT structure destroyed and tables number is 0
   query.Prepare(query_select);
   query.Next();
-  ASSERT_EQ(25, query.GetInteger(0));
+  ASSERT_EQ(kPolicyTablesNumber, query.GetInteger(0));
```

Comment:
@LuxoftAKutsan This magic number comes from the kCreateSchema query string, see [sql_pt_queries.cc](https://github.com/smartdevicelink/sdl_core/blob/feature/extended_policy_functionality/src/components/policy/policy_regular/src/sql_pt_queries.cc). Before changes tables number was 25 (just search for 'CREATE TABLE') and after changes: +2.